third_party/move/move-model/src/model.rs

```diff
@@ -2901,6 +2898,15 @@ pub struct FunctionData {
     pub(crate) transitive_closure_of_called_funs: RefCell<Option<BTreeSet<QualifiedId<FunId>>>>,
 }
 
+/// Kind of a function,
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub enum FunctionKind {
+    Regular,
+    Inline,
+    Macro,
```

Comment:
does macro refer to a function in a parameter? it doesn't appear used, and I'm not sure how this would be instantiated in the model.

Reply:
Good catch, we don't need this anymore as macros like `assert!` are now expanded during model construction time. Will resolve in the next PR.